src/plugins/aragon/index.ts

```diff
@@ -1,5 +1,113 @@
-import { sendTransaction } from '../../utils';
-import abi from './DisputableDelay.json';
+import BN from 'bn.js';
+import { keccak256 } from '@ethersproject/keccak256';
+import { toUtf8Bytes } from '@ethersproject/strings';
+import { sendTransaction, subgraphRequest } from '../../utils';
+import abi from './GovernQueue.json';
+
+const ARAGON_SUBGRAPH_URL = {
+  4: 'https://api.thegraph.com/subgraphs/name/evalir/aragon-govern-rinkeby'
```

Comment:
Let's change this to https://thegraph.com/explorer/subgraph/aragon/aragon-govern-rinkeby

For mainnet, we can use https://thegraph.com/explorer/subgraph/aragon/aragon-govern-mainnet